spk/salt-minion/Makefile

```diff
@@ -25,35 +25,69 @@ SERVICE_USER  = auto
 SERVICE_SETUP = src/service-setup.sh
 STARTABLE     = yes
 
+include ../../mk/spksrc.common.mk
+
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+CONF_DIR      = src/conf/
+endif
+
 SPK_USR_LOCAL_LINKS  = bin:env/bin/salt-call
 SPK_USR_LOCAL_LINKS += bin:env/bin/salt-minion
 SPK_USR_LOCAL_LINKS += bin:env/bin/salt-proxy
 SPK_USR_LOCAL_LINKS += bin:env/bin/spm
 
 USE_ALTERNATE_TMPDIR = 1
 
+# [cryptography]
+DEPENDS += cross/openssl3
```

Comment:
similar to duplicity PR, openssl3 is not necessarely only needed for cryptography.  I suggest removing the comment.

Reply:
Don't we need the `cross/openssl3` for the cryptography wheel to build? That was the reason for the comment.